__tests__/app.test.js

```diff
@@ -109,4 +109,43 @@ describe("All requests", () => {
                 });
         });
     });
+    describe("GET /api/articles/:article_id/comments", () => {
+        it("200: and with list of comment which they have article_id ", () => {
+            return request(app)
+                .get("/api/articles/9/comments")
+                .expect(200)
+                .then(({ body }) => {
+                    const comments = body.comments;
+                    expect(comments).toHaveLength(2);
+                    expect(comments).toBeSortedBy("created_at", {
+                        descending: false,
+                    });
+                    comments.forEach((comment) => {
+                        expect(comment).toHaveProperty("comment_id", expect.any(Number));
+                        expect(comment).toHaveProperty("votes", expect.any(Number));
+                        expect(comment).toHaveProperty("created_at", expect.any(String));
+                        expect(comment).toHaveProperty("author", expect.any(String));
+                        expect(comment).toHaveProperty("body", expect.any(String));
+                        expect(comment).toHaveProperty("article_id", expect.any(Number));
```

Comment:
you know it's going to be article id 9 so I would test for that instead of any number